packages/polyserve/src/args.ts

```diff
@@ -36,6 +36,14 @@ export const args: ArgDescriptor[] = [
     type: String,
     defaultValue: 'auto',
   },
+  {
+    name: 'do-not-compile',
+    description: 'Disables compilation for file(s) matching given pattern. ' +
```

Comment:
Can we clarify what we mean by compilation? As currently written, it bypasses ES5 compilation, but also name-to-path transformation and AMD transformation. compile-middleware is already probably too aggressive in this regard, since the `compile=never` returns `false` from `shouldTransform` in the same way.